### PR TITLE
SEO-185494-MK-.NET-MAUI-Scheduler-Interlinking

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @Girija-Rajendran 
Please review and merge the PRs,

|Title|Description|
|--|--|
|Category|Money Keyword Interlinking .NET MAUI Scheduler|
|Interlinking PR|https://github.com/syncfusion-content/maui-docs/pull/2509|
|Content Task|https://dev.azure.com/Syncfusion-General/Content/_workitems/edit/767363|
|Sharepoint| [SharePoint](https://syncfusion.sharepoint.com/sites/GH/SEO/SitePages/,.aspx?OR=Teams-HL&CT=1691733172385&clickparams=eyJBcHBOYW1lIjoiVGVhbXMtRGVza3RvcCIsIkFwcFZlcnNpb24iOiIyNy8yMzA3MTMyNzExNiIsIkhhc0ZlZGVyYXRlZFVzZXIiOmZhbHNlfQ%3D%3D)|

|Change made |Reason for change|
|---|---|
|Changed meta element to upper casesimage| For proper headings alignments|
|Modify alt text and images with platform and control name under the date-navigation-restriction and day-week-view featureimage| To define the alt text content and proper image name|
|Renamed images with platform and control name under the date-navigation-restriction and day-week-view featureimage| To define the alt text content properly and avoid redirection issues, hence improving search ranking|
|renamed alt text and images with platform and control name under resource view| To define the alt text content properly hence improve search ranking|
|Modified h1 and title content in caps under Load on demand and resource-view featureimage| For headings alignment|
|Interlinked keyword under Load on demand featureimage|For easy page view|
|added Dot(.) in alt textimage| To helps define the alt text content properly|

Regards,
Edith.

### CODE REVIEW

File: MAUI/Scheduler/day-week-views.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ this.Content = scheduler;
 {% endhighlight %}
 {% endtabs %}
 
-![visible-days-count-in-maui-scheduler-days-view](images/day-week-views/number-of-visibledays-in-maui-scheduler.png)
```
Comment (Girija-Rajendran) [inline MAUI/Scheduler/day-week-views.md:47]: Is this reviewed content?

**Comment by Girija-Rajendran** [comment]
Hi @Edithabich, 
Still, descriptions are not valid. Can you check and modify it?

File: MAUI/Scheduler/day-week-views.md (hunk 1 above)
Comment (Edithabich) [reply MAUI/Scheduler/day-week-views.md:47]: I only removed the hyphen in the alt text. 
image

File: MAUI/Scheduler/resource-view.md [hunk 2]
```diff
@@ -325,7 +325,7 @@ private ObservableCollection<SchedulerTimeRegion> GetTimeRegion()
 {% endhighlight %}
 {% endtabs %}
 
-![resource-view-special-timeregion-maui-scheduler](images/resource-view/special-regions-resources-in-maui-scheduler.png)
+![Resource view special timeregion in .NET MAUI Scheduler.](images/resource-view/net-maui-scheduler-special-regions-resources.png)
```
Comment (Girija-Rajendran) [inline MAUI/Scheduler/resource-view.md:328]: Is this meaningful? Please check all images.
image

**Review by Girija-Rajendran — CHANGES_REQUESTED** [review]

File: MAUI/Scheduler/resource-view.md [hunk 3]
```diff
@@ -108,12 +108,12 @@ this.Scheduler.AppointmentsSource = appointment;
...
+![Resource sharing in .NET MAUI Scheduler.](images/resource-view/net-maui-scheduler-resource-sharing.png)
 
 
 ## Business object binding for resources
 
-The Schedule supports full data binding to [Resources](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Scheduler.SchedulerResourceView.html#Syncfusion_Maui_Scheduler_SchedulerResourceView_Resources) in the `SchedulerResourceView` class. Specify the [SchedulerResourceMapping](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Scheduler.SchedulerResourceMapping.html#properties) property of the `SchedulerResourceView` class to map the custom properties to the schedule resource.
+The Schedule supports full data binding to [Resources](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Scheduler.SchedulerResourceView.html#Syncfusion_Maui_Scheduler_SchedulerResourceView_Resources) in the `SchedulerResourceView` class. Specify the [SchedulerResourceMapping](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Scheduler.SchedulerResourceMapping.html) property of the `SchedulerResourceView` class to map the custom properties to the schedule resource.
```
[the remark this answers is not in the document]
Comment (Edithabich) [reply MAUI/Scheduler/resource-view.md]: Reverted the changes

File: MAUI/Scheduler/resource-view.md (hunk 2 above)
Comment (Edithabich) [reply MAUI/Scheduler/resource-view.md]: Provide proper image names

**Review by Girija-Rajendran — APPROVED** [review]
fine

**Pull request merged**